gemrb/plugins/SDLVideo/CMakeLists.txt

```diff
@@ -2,11 +2,12 @@ INCLUDE_DIRECTORIES( ${SDL_INCLUDE_DIR} )
 
 SET(COMMON_FILES COCOA SDLVideo.cpp SDLSurfaceSprite2D.cpp)
 IF(SDL_BACKEND STREQUAL "SDL2")
-	IF(USE_OPENGL)
-		ADD_GEMRB_PLUGIN( SDLVideo ${COMMON_FILES} SDL20Video.cpp SDL20GLVideo.cpp GLSLProgram.cpp Matrix.cpp GLTextureSprite2D.cpp GLPaletteManager.cpp)
-		TARGET_LINK_LIBRARIES( SDLVideo ${SDL_LIBRARY} ${OPENGL_LIBRARY} ${GLEW_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT} ${COCOA_LIBRARY_PATH})
-		IF(MINGW)
-			TARGET_LINK_LIBRARIES( SDLVideo imm32 winmm version)
+	IF(OPENGL_BACKEND STREQUAL "1")
+		ADD_GEMRB_PLUGIN( SDLVideo ${COMMON_FILES} SDL20Video.cpp GLSLProgram.cpp)
+		IF(APPLE)
+			TARGET_LINK_LIBRARIES( SDLVideo ${SDL_LIBRARY} ${OPENGL_LIBRARY} ${CMAKE_THREAD_LIBS_INIT} ${COCOA_LIBRARY_PATH})
+		ELSE()
+			TARGET_LINK_LIBRARIES( SDLVideo ${SDL_LIBRARY} ${OPENGL_LIBRARY} ${GLEW_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT} ${COCOA_LIBRARY_PATH})
```

Comment:
Can COCOA_LIBRARY_PATH ever be non-empty here? I suggest removal if not.

Reply:
Yes, never made sense I guess.